.github/workflows/dotnet.yml

```
@@ -23,6 +23,9 @@ jobs:
     - uses: actions/setup-dotnet@v1
       with:
           dotnet-version: 6.0.x
+    - uses: actions/setup-dotnet@v1
+      with:
+          dotnet-version: 7.0.x

```

FYI, it supports multiple versions (at least in `v3`, I'm not that sure about `v1`).
```suggestion
          dotnet-version: |
            6.0.x
            7.0.x
```